test/components/labelTests.ts

```diff
@@ -1,198 +1,217 @@
+    });
+
+    it("errors on negative padding", () => {
+      const testLabel = new Plottable.Components.Label("testing label");
+      (<any> assert).throws(() => testLabel.padding(-10), Error, "Cannot be less than 0", "error on negative input");
+    });
   });
 
-  it("Label angle can be changed after label is created", () => {
-    let svg = TestMethods.generateSVG(400, 400);
-    let label = new Plottable.Components.AxisLabel("CHANGING ORIENTATION");
-    label.renderTo(svg);
-
-    let content = (<any> label)._content;
-    let text = content.select("text");
-    let bbox = Plottable.Utils.DOM.elementBBox(text);
-    assert.closeTo(bbox.height, label.height(), 1, "label is in horizontal position");
-
-    label.angle(90);
-    text = content.select("text");
-    bbox = Plottable.Utils.DOM.elementBBox(text);
-    TestMethods.assertBBoxInclusion((<any> label)._element.select(".bounding-box"), text);
-    assert.closeTo(bbox.height, label.width(), window.Pixel_CloseTo_Requirement, "label is in vertical position");
-
-    svg.remove();
-  });
-
-  it("padding reacts well under align", () => {
-    let svg = TestMethods.generateSVG(400, 200);
-    let testLabel = new Plottable.Components.Label("testing label").padding(30).xAlignment("left");
-    let longLabel = new Plottable.Components.Label("LONG LABELLLLLLLLLLLLLLLLL").xAlignment("left");
-    let topLabel = new Plottable.Components.Label("label").yAlignment("bottom");
-    new Plottable.Components.Table([[topLabel], [testLabel], [longLabel]]).renderTo(svg);
-
-    let testTextRect = (<any> testLabel)._element.select("text").node().getBoundingClientRect();
-    let longTextRect = (<any> longLabel)._element.select("text").node().getBoundingClientRect();
-
-    assert.closeTo(testTextRect.left, longTextRect.left + 30, 2, "left difference by padding amount");
-
-    testLabel.xAlignment("right");
-
-    testTextRect = (<any> testLabel)._element.select("text").node().getBoundingClientRect();
-    longTextRect = (<any> longLabel)._element.select("text").node().getBoundingClientRect();
-
-    assert.closeTo(testTextRect.right, longTextRect.right - 30, 2, "right difference by padding amount");
-
-    testLabel.yAlignment("bottom");
-
-    testTextRect = (<any> testLabel)._element.select("text").node().getBoundingClientRect();
-    longTextRect = (<any> longLabel)._element.select("text").node().getBoundingClientRect();
-
-    assert.closeTo(testTextRect.bottom, longTextRect.top - 30, 2, "vertical difference by padding amount");
-
-    testLabel.yAlignment("top");
-
-    testTextRect = (<any> testLabel)._element.select("text").node().getBoundingClientRect();
-    let topTextRect = (<any> topLabel)._element.select("text").node().getBoundingClientRect();
-
-    assert.closeTo(testTextRect.top, topTextRect.bottom + 30, 2, "vertical difference by padding amount");
-    svg.remove();
-  });
+  describe("TitleLabel", () => {
+    it("generates standard text title label properly", () => {
```

Comment:
It looks like this test checks two things:
* That the appropriate CSS class is applied.
* That the text rendered correctly.

Since checking correct rendering properly belongs in the superclass' tests, can we rewrite this test to check for only the CSS class?